src/getThreadList.js

```diff
@@ -4,23 +4,32 @@ var utils = require("../utils");
 var log = require("npmlog");
 
 module.exports = function(defaultFuncs, api, ctx) {
-  return function getThreadList(start, end, callback) {
+  return function getThreadList(start, end, isPending, callback) {
```

Comment:
Can we make the `pending` argument option so that we don't have to break backward compatibility?